### DISCUSSION TITLE
https://loco.rs/ is Inaccessible ？

### DISCUSSION THREAD

**Discussion by fan-tastic-z** [loco-rs/loco#147] [Q&A] [answered]
https://loco.rs/ seems to be inaccessible

**Comment by jondot** [comment, marked as answer]
Yes, thanks!
There is an issue with the registrar, I've already requested a repair which was completed successfully. Might take some more time for DNS to propagate

**Reply by fan-tastic-z** [reply]
Yes, it's fine now
